proofs/tree.go

```diff
@@ -223,9 +233,20 @@ func ValueToString(value interface{}) (s string, err error) {
 
 		// Check if elem is a zero value, return empty string if it is.
 		if elem == reflect.Zero(reflect.TypeOf(elem)) {
-			return "", nil
+			return nil
 		}
-		value = elem.Interface()
+		return  elem.Interface()
```

Comment:
I'm thinking we could do away with all these error checks with

1. Call `Elem()` -> value
2. Check for `!value.IsValid()` for errors